docker/Dockerfile.deps

```diff
@@ -30,6 +30,20 @@ ENV DALI_DEPS_REPO=${DALI_DEPS_REPO:-https://github.com/NVIDIA/DALI_deps}
 ARG DALI_DEPS_VERSION_SHA
 ENV DALI_DEPS_VERSION_SHA=${DALI_DEPS_VERSION_SHA}
 
+# Clang, build it before deps as deps changes more frequently
```

Comment:
For this to be true, shouldn't we move it above the `ARG DALI_DEPS_VERSION_SHA` and `ENV ...`?

Reply:
True, missed that. Fixed.